todo-service/src/main/java/com/brtvsk/todoservice/controller/TodoController.java

```diff
@@ -0,0 +1,68 @@
+package com.brtvsk.todoservice.controller;
+
+import com.brtvsk.todoservice.exception.TodoNotFoundException;
+import com.brtvsk.todoservice.model.dto.OptionalRequestTodoDto;
+import com.brtvsk.todoservice.model.dto.RequestTodoDto;
+import com.brtvsk.todoservice.model.dto.ResponseTodoDto;
+import com.brtvsk.todoservice.service.TodoService;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("api/todo")
+public class TodoController {
+
+    private final TodoService todoService;
+
+    public TodoController(final TodoService todoService) {
+        this.todoService = todoService;
+    }
+
+    @GetMapping
+    public List<ResponseTodoDto> all(@RequestParam(name = "done") Optional<Boolean> done) {
```

Comment:
Check it twice, but since 4.5 and java 9 `@RequestParam` isn't necessary if param and method parameter has the same name.

Reply:
Removed @RequestParam